raw.go

```diff
@@ -31,3 +31,32 @@ type RawSpan struct {
 	// The span's associated baggage.
 	Baggage map[string]string // initialized on first use
 }
+
+// SpanDetail summarizes the identifying fields of a Span, for
+// diagnostic and implementation-specific purposes.
+type SpanDetail struct {
+	// Context contains the span identifiers and whether it is
+	// traced.
+	Context
+
+	// Operation names the work done by this span instance.
+	Operation string
+
+	// Start indicates when the span began.
+	Start time.Time
+}
+
+// GetSpanDetail returns the SpanDetail and true if `span` is a
+// basictracer span, or false if `span` is not a basictracer
+// span.
+func GetSpanDetail(span opentracing.Span) (SpanDetail, bool) {
```

Comment:
given my proposal above, the `bool` return value would just be the second return value in the cast:

```
basicSpan, ok := otSpan.(basictracer.BasicSpan)
```

`</obvious>`